pkgs/applications/audio/pd-plugins/timbreid/default.nix

```diff
@@ -1,30 +1,45 @@
-{ stdenv, fetchurl, unzip, puredata }:
+{ stdenv, fetchurl, unzip, puredata, fftwSinglePrec }:
 
 stdenv.mkDerivation rec {
-  version = "0.6.0";
+  version = "0.7.0";
   name = "timbreid-${version}";
 
   src = fetchurl {
     url = "http://williambrent.conflations.com/pd/timbreID-${version}-src.zip";
-    sha256 = "02rnkb0vpjxrr60c3hryv7zhyjpci2mi9dk27kjxpj5zp26gjk0p";
+    sha256 = "14k2xk5zrzrw1zprdbwx45hrlc7ck8vq4drpd3l455i5r8yk4y6b";
   };
 
-  buildInputs = [ unzip puredata ];
+  buildInputs = [ unzip puredata fftwSinglePrec ];
+
+  makeFlags = [
+    "CFLAGS= -I${puredata}/include/pd -Wall -W -g “-fPIC” --enable-float"
+    "LIBDIR=$(out)/lib"
+  ];
 
   unpackPhase = ''
+    mkdir source
+    cd source
     unzip $src
-    mv timbreID-0.6.0-src/tID/* .
-    rm -rf timbreID-0.6.0-src/tID/
-    rm -rf timbreID-0.6.0-src/INSTALL.txt
   '';
 
+  buildPhase = ''
+    make tIDLib.o all
+ '';
+
   installPhase = ''
     mkdir -p $out/
     cp -r *.pd $out/
     cp -r *.pd_linux $out/
-    cp -r *.wav $out/
+    cp -r audio/ $out/
+    cp -r data/ $out/
+    cp -r doc/ $out/
   '';
 
+  postFixup = ''
+    mv $out/share/doc/ $out/
+    rm -rf $out/share/
+    '';
+
```

Comment:
This is not needed. The default fixup phase moves `doc` in the right place`.

Reply:
it's actually the fixup phase that moves it from the right place to the wrong one, and this moves it back

Reply:
Then fixup is right.

Reply:
@orivej I realize that normally docs go in $out/share/doc, but in this case some of the help patches point to stuff in $out/doc, so I moved it.